src/mount.azfs

```diff
@@ -446,21 +291,23 @@ done
  
 nfs_host=$(get_host_from_share "$1") 
 
- 
+
 if ! is_valid_blob_fqdn $nfs_host; then 
```

Comment:
is_valid_blob_fqdn and many other functions need to be added to this script

Reply:
do you mean those already added functions need to be updated? Since this function is_valid_blob_fqdn is already there.